### DISCUSSION TITLE
New release please

### DISCUSSION THREAD

**Discussion by hweckstr** [custom-components/nordpool#318] [General]
As the Finnish tax changed from 10% to 24% on 1st of May, and the change is already in the master. When can we expect a new release to this marvellous HA integration. I will buy a coffee for a quick release :-)

**Comment by Hellowlol** [comment]
Hi,

i will get it done tomorrow. In the meantime you can set your own vat using the example https://github.com/custom-components/nordpool#example-1-overhead-per-kwh
